modFastVM/src/org/aion/solidity/Compiler.java

```diff
@@ -103,6 +118,27 @@ public Result compile(byte[] source, boolean optimize, boolean combinedJson, Opt
         }
     }
 
+    private File extractZip(byte[] source, String tempDirName) throws IOException {
+        File tempDir = new File(tempDirName);
+        tempDir.mkdir();
```

Comment:
should check the directory has been created. then go next step.

Reply:
Fixed